components/molecule/imageEditor/src/styles/index.scss

```diff
@@ -38,4 +38,32 @@ $base-class: '.sui-MoleculeImageEditor';
       width: $w-molecule-image-editor-label-text;
     }
   }
+
+  &-helpText {
+    margin-top: $m-xxxl;
+  }
+
+  &-rowLayout {
```

Comment:
IMHO this would be a modifier and hence should follow the `--rowLayout` format following the [SUIT convention](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentname--modifiername), what do you think?